examples/EXAMPLES.md

```diff
@@ -20,3 +20,69 @@ or similar. Hyperparameters can be easily changed by providing the arguments to
 ```
 python sac frames_per_batch=63
 ```
+# Results
+
+## Half Cheetah
+Here we can see some results for the SAC and REDQ algorithm on the Half Cheetah environement.
```

Comment:
What about discarding this sentence, which does not add much?

Reply:
I've changed it such as it's only at the begining of the results section and not in every example